### PR TITLE
Change yahboom documentation to modular.

### PR DESCRIPTION
**Pull request by randhid**
# Description

We're kicking the yahboom arm out of the rdk. I've changed some of the wording and removed the attributes (there are none now https://github.com/viam-labs/yahboom).

I don't have access to viam-labs right now so can't upload it ... boo! So I've used my name as a placeholder for the namespace for now.

We can also not document this arm - I don't think we're uploading it to the viam namespace. 

Should I keep the state of the module (only supports joint position movement for now) in the readme?

Thanks.

### CODE REVIEW

File: docs/components/arm/yahboom-dofbot.md [hunk 1]
```diff
@@ -3,20 +3,22 @@ title: "Configure a yahboom-dofbot Arm"
 linkTitle: "yahboom-dofbot"
 weight: 50
 type: "docs"
-description: "Configure a yahboom-dofbot arm."
+draft: "true"
+description: "Configure a yahboom dofbot modular arm."
```
Comment (JessamyT) [inline docs/components/arm/yahboom-dofbot.md:7]:
```suggestion
description: "Configure a Yahboom DOFBOT modular arm."
```
If it's not the model name anymore -> sentence case

File: docs/components/arm/yahboom-dofbot.md [hunk 2]
```diff
@@ -3,20 +3,22 @@ title: "Configure a yahboom-dofbot Arm"
...
 images: ["/icons/components/arm.svg"]
 tags: ["arm", "components"]
 ---
 
-Configure a `yahboom-dofbot` arm to add a [Yahboom DOFBOT](https://category.yahboom.net/collections/r-robotics-arm) to your robot:
+Viam provides a `dofbot` arm from the manufacturer `yahboom` as a modular resource. To add a [Yahboom DOFBOT](https://category.yahboom.net/collections/r-robotics-arm) to your robot:
```
Comment (JessamyT) [inline docs/components/arm/yahboom-dofbot.md:12]:
```suggestion
Viam supports the [Yahboom DOFBOT](https://category.yahboom.net/collections/r-robotics-arm) arm as a modular resource.
Configure a `dofbot` arm to add it to your robot:
```

File: docs/components/arm/yahboom-dofbot.md [hunk 3]
```diff
@@ -3,20 +3,22 @@ title: "Configure a yahboom-dofbot Arm"
...
 
 {{< tabs >}}
 {{% tab name="Config Builder" %}}
 
 Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
-Click on the **Components** subtab and click **Create component**.
-Select the `arm` type, then select the `yahboom-dofbot` model.
+Click on the **Components** subtab and find the **Create component** pane.
```
Comment (JessamyT) [inline docs/components/arm/yahboom-dofbot.md:18]:
```suggestion
Click on the **Components** subtab and click **Create component**.
```
Per recent UI change

File: docs/components/arm/yahboom-dofbot.md [hunk 4]
```diff
@@ -3,20 +3,22 @@ title: "Configure a yahboom-dofbot Arm"
...
+Select the `arm` type, then select the `rand:yahboom:dofbot` modular resource.
 Enter a name for your arm and click **Create**.
+Note: There are not attributes for this modular arm.
```
Comment (JessamyT) [inline docs/components/arm/yahboom-dofbot.md:21]:
```suggestion

There are no attributes available for this modular arm.
```

File: docs/components/arm/yahboom-dofbot.md [hunk 5]
```diff
@@ -29,13 +31,10 @@ Edit and fill in the attributes as applicable.
 {
   "components": [
     {
-      "model": "yahboom-dofbot",
+      "model": "dofbot",
       "name": "<your-arm-name>",
```
Comment (JessamyT) [inline docs/components/arm/yahboom-dofbot.md:35]:
```suggestion
      "name": "<your-arm-name>",
      "model": "dofbot",
```
[nit] For consistency with PR 1920

**Review by JessamyT — COMMENTED** [review]
Only reviewing because I was just working on this file in #1920--I will take it out of that PR to avoid conflicts. Once this module becomes available, if Naomi decides this should stay in the documentation somewhere, we'll have to update the UI image. I'm not creating a ticket for that yet since IDK what will be decided as to the fate of this doc.

**Comment by randhid** [comment]
> Only reviewing because I was just working on this file in #1920--I will take it out of that PR to avoid conflicts. Once this module becomes available, if Naomi decides this should stay in the documentation somewhere, we'll have to update the UI image. I'm not creating a ticket for that yet since IDK what will be decided as to the fate of this doc.

No worries - if you and naomi decide you'd like to take this over, I'm also happy with that! We lost a docs ticket when this was taken out of the modular resources epic. I'm about to merge the deletion in  - https://github.com/viamrobotics/rdk/pull/3003, so I thought I'd be responsible and change this file too.

File: docs/components/arm/yahboom-dofbot.md [hunk 6]
```diff
@@ -3,20 +3,22 @@ title: "Configure a yahboom-dofbot Arm"
 linkTitle: "yahboom-dofbot"
 weight: 50
 type: "docs"
-description: "Configure a yahboom-dofbot arm."
+draft: "true"
```
Comment (npentrel) [inline docs/components/arm/yahboom-dofbot.md:6]: does it need to be a draft?

**Review by npentrel — APPROVED** [review]
This looks good to me but I think we should also add a link to the github repo for the modular resource. Please do accept Jessamy's comments though

File: docs/components/arm/yahboom-dofbot.md (hunk 2 above)
Comment (npentrel) [reply docs/components/arm/yahboom-dofbot.md:12]:
```suggestion
Viam supports the [Yahboom DOFBOT](https://category.yahboom.net/collections/r-robotics-arm) arm as a [modular resource](https://github.com/viam-labs/yahboom).
Configure a `dofbot` arm to add it to your robot:
```

File: docs/components/arm/yahboom-dofbot.md (hunk 6 above)
Comment (randhid) [reply docs/components/arm/yahboom-dofbot.md:6]: Until I publish the module - I'd rather yes - also I wasn't sure if you were documenting viam-labs modules, which are meant to be less well-vetted than the ones in the viam namespace.

**Comment by randhid** [comment]
@JessamyT Let me know if it's easier for you to take over these changes and include them in your pr.

**Pull request merged**